rewrite-javascript/src/main/java/org/openrewrite/javascript/cleanup/UseCaseFallThrough.java

```diff
@@ -71,7 +71,7 @@ public J.Switch visitSwitch(J.Switch switch_, ExecutionContext ctx) {
                 }
 
                 s = s.withCases(s.getCases().withStatements(ListUtils.flatMap(s.getCases().getStatements(), it -> {
-                    if (it instanceof J.Case && changeCondition(((J.Case) it).getExpressions())) {
+                    if (it instanceof J.Case && changeCondition(((J.Case) it).getCaseLabels())) {
```

Comment:
Same concern as on the Python counter part: I'm not sure these replacement fields are populated here already.
- https://github.com/openrewrite/rewrite-python/pull/136?email_source=slack

Reply:
Olegchanged the javascript remoting parser to use the CaseLabels back when it was introduced